protonmail/messages.go

```diff
@@ -15,6 +15,9 @@ import (
 	"golang.org/x/crypto/openpgp/packet"
 )
 
+// PageSize are enforced by the ProtonMail server for all message operations
+const PageSize = 150
```

Comment:
Maybe rename to `MaxMessagePageSize`? This limit applies to messages only.

Reply:
Agreed. Changed so.